### DISCUSSION TITLE
Define nextcloud storage

### DISCUSSION THREAD

**Discussion by joaopinto14** [nextcloud/all-in-one#913] [Questions] [answered]
Is it possible to define the location where the data that is placed in the cloud is stored?
On web installation (script, zip download) it's possible, but with the all-in-one docker image I can't, at least not easily.
Someone help me?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-change-the-default-location-of-nextclouds-datadir and https://github.com/nextcloud/all-in-one#how-to-store-the-filesinstallation-on-a-separate-drive

**Reply by rkbest13** [reply]
Thanks @szaimen I did defined the data directory in the stack as in the github page but my data directory was empty when the CT was running. Here is my compose. wondering what is wrong here. 
`
![image](https://user-images.githubusercontent.com/64113851/178303980-19c0c990-7a9e-4d91-9f19-8a80963d98eb.png)

`
